subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -2051,20 +2051,17 @@ uint16_t ull_conn_lll_max_tx_octets_get(struct lll_conn *lll)
 
 /**
  * @brief Initialize pdu_data members that are read only in lower link layer.
+ *        Fields that are modified after static initialization are only
```

Comment:
Hm... I'm not sure if I get the new part of description correctly.

The function purpose is to initialize members that should not be modified in LLL.
The CP bit and RFU should not be modified when direction finding is enabled.
If PDU is fragmented these fields are initialized in new fragments, so it does not apply to original PDU header.

I suggest to leave original comment only.

Reply:
Removed.